### PR TITLE
Make error msgs more go style by uncapitalize them

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: cmd/txt-prov/cmd.go [hunk 1]
```diff
@@ -214,21 +214,17 @@ func (p *platProvCmd) Run(ctx *context) error {
...
 		}
 		if lock {
 			return fmt.Errorf("NVRAM is locked, please disable Intel TXT or any firmware TPM driver")
 		}
 		lcp, err := loadConfig(p.Config)
 		if err != nil {
-			return fmt.Errorf("Couldn't parse LCP config file: %v", err)
+			return fmt.Errorf("couldn't parse LCP config file: %v", err)
 		}
-		passHash, err := readPassphraseHashTPM20()
```
Comment (PatrickRudolph) [inline cmd/txt-prov/cmd.go:226]: change in functionality

File: pkg/intel/metadata/manifest/crypto_routines.go [hunk 2]
```diff
@@ -132,6 +132,6 @@ func GetAlgFromString(name string) (Algorithm, error) {
 	case "SM2":
 		return AlgSM2, nil
 	default:
-		return AlgNull, fmt.Errorf("Algorihm name provided unknown")
+		return AlgNull, fmt.Errorf("algorihm name provided unknown")
```
Comment (PatrickRudolph) [inline pkg/intel/metadata/manifest/crypto_routines.go:135]: algorithm

**Review by PatrickRudolph — COMMENTED** [review]
You changed more than capital letters.
Maybe: Fix linting errors found by .... and return errors in more cases.

File: cmd/txt-prov/cmd.go (hunk 1 above)
Comment (ChriMarMe) [reply cmd/txt-prov/cmd.go:226]: check

File: pkg/intel/metadata/manifest/crypto_routines.go (hunk 2 above)
Comment (ChriMarMe) [reply pkg/intel/metadata/manifest/crypto_routines.go]: check

**Comment by ChriMarMe** [comment]
Linting errors are related to hwapi which will be removed with #279

**Review by PatrickRudolph — APPROVED** [review]

**Review by rihter007 — APPROVED** [review]

**Review by xaionaro — APPROVED** [review]

**Pull request merged**